app/controllers/api/v2/hosts_controller.rb

```diff
@@ -250,14 +250,14 @@ def boot
         render_message(e.to_s, :status => :unprocessable_entity)
       end
 
-      api :POST, "/hosts/facts", N_("Upload facts for a host, creating the host if required")
+      api :POST, "/hosts/facts", N_("Upload facts for a host, creating the host if required (bypasses username/password authentication and validates trusted hosts setting or known smart proxies based on client cert CN or PTR record)")
```

Comment:
1. This seems unrelated to the ticket?
1. Use a description field for this, not the method title.
1. Also applies to other APIs such as report uploads, those should be updated at the same time.